azurerm/resource_arm_log_analytics_linked_service.go

```diff
@@ -0,0 +1,232 @@
+package azurerm
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/Azure/azure-sdk-for-go/services/preview/operationalinsights/mgmt/2015-11-01-preview/operationalinsights"
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/hashicorp/terraform/helper/validation"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/azure"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/suppress"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/tf"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
+)
+
+/*
+TODO: refactor this:
+
+ * resource_group_name/workspace_name can become case-sensitive
```

Comment:
Should we put `in 2.0` here to make sure it gets done?

Reply:
I'm not so concerned about that since we'll be taking a look through the TODO's as a part of 2.0 anyway, imo?